builder/processor-tools/src/main/java/io/helidon/builder/processor/tools/BeanUtils.java

```diff
@@ -110,10 +123,21 @@ public static boolean validateAndParseMethodName(String methodName,
      * @return true if it appears to be a reserved word
      */
     public static boolean isReservedWord(String word) {
-        word = word.toLowerCase();
-        return word.equals("class") || word.equals("interface") || word.equals("package") || word.equals("static")
-                || word.equals("final") || word.equals("public") || word.equals("protected") || word.equals("private")
-                || word.equals("abstract");
+        word = word.toUpperCase();
+        return word.equals(KIND_CLASS) || word.equals(KIND_INTERFACE) || word.equals(KIND_PACKAGE) || word.equals(KIND_ENUM)
+                || word.equals(MODIFIER_STATIC) || word.equals(MODIFIER_FINAL) || word.equals(MODIFIER_PUBLIC)
+                || word.equals(MODIFIER_PROTECTED) || word.equals(MODIFIER_PRIVATE) || word.equalsIgnoreCase(KIND_RECORD)
+                || word.equals(MODIFIER_ABSTRACT);
+    }
```

Comment:
This would be more readable if every equality check is on the separate line.

Or even possibly to create some Set cache, which would contain all of these and instead of asking each of the key separately, there would be just a simple contains check to the Set.

Reply:
next push.